blog-posts/howto-blog-post.md

```diff
@@ -5,6 +5,7 @@ title: "How to do a blog post"
 featuredImage: images/space.jpg
 author: "Georgios Kaleadis & Fabian Dietenberger"
 authorSummary: "CTO and Senior Developer at Satellytes"
+teaserText: "Learn how to properly write a blog post – in terms of formatting – so that everything is rendered properly & our blog looks super shiny."
```

Comment:
FYI You don't need quotes for the content in yaml and I mix this up also regularly. You might have copied another yaml line for me here because it's good in the other files 😅